test/run_pass/test_fs_exists.js

```diff
@@ -25,6 +25,18 @@ var assert = require('assert');
 }
 
 {
+  var filePath = "";
+
+  fs.exists(filePath);
```

Comment:
The test case isn't completed with the comparison between expectation and result using assert. This problem can be found in other files as well as here.